docs/src/content/docs/fr/getting-started.mdx

```diff
@@ -45,26 +46,71 @@ Essayez Starlight dans votre navigateur :
 [ouvrir le modèle sur StackBlitz](https://stackblitz.com/github/withastro/starlight/tree/main/examples/basics).
 :::
 
-## Créer du contenu avec Starlight
+### Démarrer le serveur de développement
 
-Starlight est conçu pour que vous puissiez ajouter du nouveau contenu, ou apporter vos fichiers existants !
+Lorsque vous travaillez localement, [le serveur de développement d'Astro](https://docs.astro.build/fr/reference/cli-reference/#astro-dev) vous permet de prévisualiser votre travail et actualise automatiquement votre navigateur lorsque vous apportez des modifications.
 
-### Formats de fichiers
+À l'intérieur du répertoire de votre projet, exécutez la commande suivante pour démarrer le serveur de développement :
 
-Starlight prend en charge la création de contenu en Markdown et MDX. (Vous pouvez ajouter la prise en charge de Markdoc en installant l'intégration expérimentale [Astro Markdoc integration](https://docs.astro.build/fr/guides/integrations-guide/markdoc/).)
+<Tabs>
+<TabItem label="npm">
 
-### Ajouter des pages
+```sh
+npm run dev
+```
 
-Ajoutez automatiquement de nouvelles pages à votre site en créant des fichiers `.md` ou `.mdx` dans `src/content/docs/`. Ajoutez des sous-dossiers pour organiser vos fichiers, et pour créer plusieurs segments de chemin :
+</TabItem>
+<TabItem label="pnpm">
 
+```sh
+pnpm dev
 ```
-src/content/docs/hello-world.md => votre-site.com/hello-world
-src/content/docs/guides/faq.md => votre-site.com/guides/faq
+
+</TabItem>
+<TabItem label="Yarn">
+
+```sh
+yarn dev
 ```
 
-### Type-safe frontmatter
+</TabItem>
+</Tabs>
+
+Un message indiquant l'URL de votre aperçu local s'affichera dans votre terminal.
+Ouvrez cette URL pour commencer à naviguer sur votre site.
+
+### Ajouter du contenu
+
+Starlight est prêt pour que vous puissiez ajouter du nouveau contenu, ou apporter vos fichiers existants !
+
+#### Formats de fichiers
+
+Starlight prend en charge la création de contenu en Markdown et MDX sans aucune configuration requise.
+Vous pouvez ajouter la prise en charge de Markdoc en installant [l'intégration Astro Markdoc](https://docs.astro.build/fr/guides/integrations-guide/markdoc/) expérimentale.
+
+#### Ajouter des pages
+
+Ajoutez de nouvelles pages à votre site en créant des fichiers `.md` ou `.mdx` dans `src/content/docs/`.
+Utilisez des sous-dossiers pour organiser vos fichiers et créer plusieurs segments de chemin.
+
+Par exemple, la structure de fichiers suivante génére des pages à `example.com/hello-world` et `example.com/guides/faq` :
```

Comment:
Decided to not translate "hello world" based on the content of [this page](https://fr.wikipedia.org/wiki/Hello_world) which indicates that we can translate it literally but it's not really used or common.